### ISSUE TITLE
[Android Bootcamp] Pre-work: ToDo App #2

### ISSUE THREAD

**Issue by nttra7** [nttra7/Android-ToDo#1]
My app is not complete but I tried my best so far. Pls review. Thanks! / cc @coderschoolreview

**Comment by coderschoolreview** [comment]
### Thank you for your Rails submission, nttra7!

:angry: Unfortunately **there are some problems** with your submission:
* I couldn't find how much time you spent on the assignment. Please include the number of hours spent on your assignment in your `README.md`.
* I couldn't find a walkthrough GIF referenced from your `README.md`. Please use [RecordIt](http://recordit.co) or [LICEcap](http://www.cockos.com/licecap/) to record an animated GIF of your submission going through the required stories, and include it in your `README.md`.
* I couldn't find a Heroku deploy link listed in your `README.md`. Please note the URL where you have deployed your app.

**Comment by harley** [comment]
Hi @nttra7, I don't see any code in your repo. Can you make sure you push your code up to GitHub?

**Comment by nttra7** [comment]
Thanks. I just pushed the code up to GitHub. Is it fine to upload the files as zip format?

**Comment by chug2k** [comment]
No, zip file is not the best. If you are familiar with terminal...

One way to reset your repo (do this very carefully):

```
cd path/to/your/app
rm -rf .git
git init 
git add .
git commit -m "Re-commiting code from scratch"
git remote add origin https://github.com/nttra7/Android-ToDo.git
git push -u origin master --force
```

### Small Explanation
- `rm -rf .git` removes any git-related information (basically, starts over)
- `git init` starts a new git repository
- `git add .` adds all the files to be "staged", so you can commit them
- `git commit -m "asdf"` saves a new commit, with the message specified after `-m`
- `git remote add origin ...` adds the github server, and names it "origin"
- `git push -u origin master --force` uploads to github. Don't use `--force` in normal situations, this is a special case because we are trying to overwrite your old code. 

If you have questions, please send an email to android {at} coderschool.vn. I can help you over screenshare, or meet you at a cafe.

**Comment by nttra7** [comment]
Thanks for your reply. I just sent a pm to android {at} coderschool.vn.
